instrumentation/aws_sdk/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History: opentelemetry-instrumentation-aws_sdk
 
+### v0.2.2 / 2022-01-26
+
+* FIXED: add missing require for aws_sdk in instrumentation all #1054
```

Comment:
Moving that FIXED to `-all`.

```suggestion
* (No significant changes)
```

The HTTP status code attribute addition we talked about in CNCF Slack was released and mentioned in the CHANGELOG for `opentelemetry-instrumentation-aws_sdk` v0.2.1 (see below outside this diff).

There _was_ [a change to this instrumentation since v0.2.1](https://github.com/open-telemetry/opentelemetry-ruby/pull/1103), but I think that falls under "no significant changes."

```
» git lg instrumentation/aws_sdk
* 4cfe0baa - (HEAD -> main, origin/main, origin/HEAD) Consistent CRLF in AWS SDK Instrumentation Changelog (#1103) (5 days ago) [Nathaniel Ruiz Nowell]
* c7a8d27d - (tag: opentelemetry-instrumentation-aws_sdk/v0.2.1) release: Release opentelemetry-instrumentation-aws_sdk 0.2.1 (#1099) (5 days ago) [github-actions[bot]]
* 89e86e34 - Add HTTP Status Code attribute to AWS SDK span (#1095) (5 days ago) [Nathaniel Ruiz Nowell]
* 9727f2ef - (tag: opentelemetry-instrumentation-aws_sdk/v0.2.0) release: Release opentelemetry-instrumentation-aws_sdk 0.2.0 (#1093) (6 days ago) [github-actions[bot]]
* 658440f2 - feat: SQS / SNS messaging attributes and context propagation (#1026) (7 days ago) [Yaniv Davidi]
```